aodndata/moorings/products_handler.py

```diff
@@ -116,6 +119,20 @@ def preprocess(self):
             product_file.publish_type = PipelineFilePublishType.HARVEST_UPLOAD
             self.file_collection.add(product_file)
 
+    def preprocess(self):
```

Comment:
just rename to ```process``` - it's making stuff in make_aggregate_timeseries.

Reply:
Again, this is overriding a method in `HandlerBase` so that it is executed at a specific step in the handler workflow. See https://github.com/aodn/python-aodncore/blob/master/HandlerBase_state_machine.png and https://aodn.github.io/python-aodncore/userdoc.html (note the documentation is a bit out of date, but the overview is still correct - for more details see the [handlerbase](https://github.com/aodn/python-aodncore/blob/master/aodncore/pipeline/handlerbase.py) code)